scripts/test-ci-all.sh

```diff
@@ -87,6 +87,8 @@ export -f cli_test_always_success
 tmpdir=$(mktemp --tmpdir -d XXXXX)
 trap 'rm -r $tmpdir' EXIT
 joblog="$tmpdir/joblog"
+trap 'if [ "$?" -ne 0 ]; then echo "##Client Database"; dbtool client list; echo "##Server Database"; dbtool server-0 list; fi' EXIT
```

Comment:
Can't really do it here. It needs to be done per test, as each test will run in it's own temp dir, etc. Probably you want make some

```
fn dump_db_content_on_termination() {
  # trap here
}
```

and call it somewhere early in every test.

Reply:
Hello @dpc, can I create `fn dump_db_content_on_termination()` at [test-ci-all.sh](https://github.com/fedimint/fedimint/blob/master/scripts/test-ci-all.sh#L25) and the call it all other function.